datalad/tests/test_annexrepo.py

```diff
@@ -0,0 +1,36 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Tests to test implementation of class AnnexRepo
+
+Note: There's not a lot to test by now.
+
+"""
+__author__ = 'Benjamin Poldrack'
+
+import os.path
+from shutil import rmtree
+
+from nose.tools import assert_raises, assert_is_instance, assert_true
+from git.exc import GitCommandError
+
+from datalad.support.annexrepo import AnnexRepo
+from datalad.tests.utils import with_tempfile, with_testrepos, assert_cwd_unchanged
+
+
+@assert_cwd_unchanged
+@with_testrepos(flavors=['local'])
+@with_tempfile
+def test_AnnexRepo(src, dst):
+
+    ar = AnnexRepo(dst, src)
+    assert_is_instance(ar, AnnexRepo, "AnnexRepo was not created.")
+    assert_true(os.path.exists(os.path.join(dst, '.git', 'annex')))
+
+    #do it again should raise GitCommandError since git will notice there's already a git-repo at that path
+    assert_raises(GitCommandError, AnnexRepo, dst, src)
```

Comment:
> That's the way it already is?!

so add another test that you could call without "src" and see no exception.  It should be ok to get AnnexRepo on top of existing/already cloned repositories... which is what would be the most frequent use case.  

As for cloning,  it might get additional bunch of options so it would be lame to channel them through all through the constructor.  But ok -- let's think about that later ... just mnake sure for now we can use it on top of existing ones, and that it is all "deterministic" (e.g. if src is provided but repository exists already - at least a warning should be out or even an exception)